gax/src/test/java/com/google/api/gax/rpc/OperationCallableImplTest.java

```diff
@@ -738,7 +748,7 @@ public void testFutureCancelInTheMiddle() throws Exception {
         mockGetOpSnapshotCallable(StatusCode.Code.OK, resultOperation);
 
     OperationSnapshot[] pollOperations = new OperationSnapshot[iterationsCount];
-    for (int i = 0; i < iterationsCount; i++) {
+    for (int i = 0; i < iterationsCount - 1; i++) {
```

Comment:
Why is this necessary? Is it just a bugfix because getOperation is already called once on line 746? Why is it only appearing now? (Or did it not appear as a failure, but you just spotted it?)

Reply:
Yes, it is just a small fix of something I noticed only once (it just prevent creating the last element tin the `pollOperations`, which will be overridden later anyways).